aten/src/ATen/native/vulkan/Vulkan.cpp

```diff
@@ -0,0 +1,1152 @@
+#ifdef USE_VULKAN
```

Comment:
I was commenting somewhere already I think - you can make file inclusion conditional in the build and then no need to have #ifdefs in every file

Reply:
Thanks, I am doing split what to compile:
in aten/src/ATen/CMakeLists.txt:

```
if(USE_VULKAN OR USE_GLES)
  set(all_cpu_cpp ${all_cpu_cpp} ${native_vulkan_cpp} ${vulkan_generated_cpp})
else()
  set(all_cpu_cpp ${all_cpu_cpp} ${native_vulkan_stub_cpp})
endif()
```
I will configure it there and remove ifdefs.